data/json/proficiencies/misc.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type": "proficiency",
+    "id": "prof_knapping",
+    "name": { "str": "Knapping" }
```

Comment:
I assume things like associated skills and a description string can come later. This PR has enough to it, I think it can wait for now.

Reply:
That was the intent, yes.